generator/config/search/equals.yaml

```diff
@@ -0,0 +1,104 @@
+# $schema: ../schema.json
+name: equals
+link: 'https://www.mongodb.com/docs/atlas/atlas-search/equals/'
+type:
+    - searchOperator
+encode: object
+description: |
+    The equals operator checks whether a field matches a value you specify.
+arguments:
+    -
+        name: path
+        type:
+            - searchPath
+    -
+        name: value
+        type:
+            - binData
+            - bool
+            - date
+            - objectId
+            - 'null'
```

Comment:
Is this necessary to overcome some YAML behavior, where `null` would be interpreted as `~`?

Reply:
Exactly, the unquoted `null` string is the NULL value in Yaml. But it seems that I can disable this behavior: https://github.com/symfony/yaml/blob/3ced3f29e4f0d6bce2170ff26719f1fe9aacc671/CHANGELOG.md#440

Let's stay safe and keep the quotes, if other scripts use this Yaml files.